src/x509.c

```diff
@@ -115,14 +115,18 @@ unsigned int wolfSSL_X509_get_extended_key_usage(WOLFSSL_X509* x509)
 /* Returns the number of X509V3 extensions in X509 object, or 0 on failure */
 int wolfSSL_X509_get_ext_count(const WOLFSSL_X509* passedCert)
 {
-    int extCount = 0;
+    int extCount = WOLFSSL_FAILURE;
```

Comment:
Using `0` as the init here seemed better. This is a count...

Reply:
done